aws/rust-runtime/aws-sig-auth/src/middleware.rs

```diff
@@ -91,27 +91,18 @@ impl MapRequest for SigV4SigningStage {
     type Error = SigningStageError;
 
     fn apply(&self, req: Request) -> Result<Request, Self::Error> {
-        req.augment(|req, config| {
+        req.augment(|mut req, config| {
             let (operation_config, request_config, creds) = signing_config(config)?;
 
             // A short dance is required to extract a signable body from an SdkBody, which
             // amounts to verifying that it a strict body based on a `Bytes` and not a stream.
             // Streams must be signed with a different signing mode. Separate support will be added for
```

Comment:
Seems like this comment needs some tweaking with the changes below?

Reply:
yep done